.github/workflows/build.yml

```diff
@@ -1,4 +1,4 @@
-on: [pull_request, push]
+on: [pull_request, push, workflow_dispatch]
```

Comment:
Is this required?

Reply:
Oops! I forgot that in there. I just removed it.